plugin/template/README.md

```diff
@@ -0,0 +1,135 @@
+# template
+
+*template* allows for dynamic responses base on the incoming query.
+
+## Syntax
+
+~~~
+template CLASS TYPE [REGEX...] {
+    [answer section]
```

Comment:
is `answer` a keyword? should this be `answer SECTION`?
(not sure how to read this yet)

Reply:
no authoritity section?

Reply:
or better `answer RR`